pkgs/development/libraries/libfprint-tod/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, libfprint
+, fetchFromGitLab
+}:
+
+# for the curious, "tod" means "Touch OEM Drivers" meaning it can load
+# external .so's.
+libfprint.overrideAttrs ({ ... }: rec {
```

Comment:
I am very confused by this.

Reply:
Ah! qyliss showed me this. `overrideAttrs` takes a functions whose only argument is the original attributes you're overriding. Often you'll see `overrideAttrs(old: { my overrides })`. Instead of calling it old, you can destructure the inputs by saying the function takes an attribute set, but the `...` means it ignores all of them.